src/View/Helper/HtmlHelper.php

```diff
@@ -156,8 +156,8 @@ public function meta(array|string $type, array|string|null $content = null, arra
                 $types['icon']['link'] = 'favicon.ico';
             }
 
-            if ($type === 'csrfToken') {
-                $types['csrfToken']['content'] = $this->_View->getRequest()->getAttribute('csrfToken');
+            if ($type === 'csrf-token') {
```

Comment:
Perhaps we should only set the token if `content` is null? That would help avoid the problem just as much as renaming the magic key does.

Reply:
Yes that's an option too.

Reply:
In hindsight I should have used  the same magic name as the tag name.